### DISCUSSION TITLE
Playlists are paused for 24000 s

### DISCUSSION THREAD

**Discussion by BuckeyePay** [Chimildic/goofy#235] [Помощь с ошибкой] [answered]
Just lately many of my playlists which formerly worked just fine are now failing due to a lengthy pause. Not sure why.

function MenBuilder() {
    let banTracks = [];
    let recentTracks = RecentTracks.get(1500);
   
    Combiner.push(banTracks, recentTracks);
          
        let mentracks = Source.getSavedAlbumTracks();

          Filter.rangeTracks(mentracks, {
          meta: {
          popularity: { min: 30, max: 99 },
    },
    });
         
        let womentracks = Source.getTracks ([ { name: 'women anthology', id: '7gdhanmCBc4VbYLTiZSJR1' },])
       
  Filter.removeTracks(mentracks, womentracks);
                           
  Filter.removeTracks(mentracks, banTracks);      
       
  Filter.matchExcept(mentracks, 'Demo');
  Filter.matchExcept(mentracks, 'Alternate');
  Filter.matchExcept(mentracks, 'Outtake');
  Filter.matchExcept(mentracks, 'Solo Piano Version');
  Filter.matchExcept(mentracks, 'Rehearsal');
  Filter.matchExcept(mentracks, 'Edit');
  Filter.matchExcept(mentracks, 'work in progress');
  Filter.matchExcept(mentracks, 'early take');
  Filter.dedupArtists(mentracks);

      Order.shuffle(mentracks)

Playlist.saveWithReplace({
        name: 'men',
        tracks: mentracks,
           });
}

the error messages.....

Операция продолжится после паузы 23993с.
5:29:10 PM	Error	
Exception: Specified sleep period exceeds maximum.
pause	@ library.gs:3531
sendPack	@ library.gs:152
fetchAll	@ library.gs:123
getAll	@ library.gs:3061
getFullObjByIds	@ library.gs:3042
cacheToFullObj	@ library.gs:2872
cache	@ library.gs:2839
getCache	@ library.gs:2830
rangeTracks	@ library.gs:1005
MenBuilder	@ men.gs:9

**Comment by Chimildic** [comment, marked as answer]
The duration of pause provide Spotify when rate limits error occurs - too many requests per short period. Usually is 1-10 seconds.

In your case looks like temporary block by Spotify due to "spam" of requests.

1. Delete all triggers
2. Wait target pause
3. Set new triggers that don't run at the same time

> trigger for `runTasks_` restore automatically after first launch any code

**Reply by BuckeyePay** [reply]
Thank you. I thought that might be the case.
